phoenicis-scripts/src/main/java/org/phoenicis/scripts/Installer.java

```diff
@@ -7,5 +7,5 @@
     /**
      * runs the installer
      */
-    void run();
+    void go();
```

Comment:
Why do you prefer `go` over `run`?

Reply:
I don't prefer `go` over `run`. Using `go` just required less changes because it has been defined in the quick script implementations. So if we want to continue using `run` we would need to change the quick scripts to provide a `run` method instead of a `go` method.

Reply:
Ok.